### PR TITLE
Make the bot less pingy and grammatically painful

### PR DESCRIPTION
**Pull request by galenguyer**
please, the commas and pings hurt me

### CODE REVIEW

**Comment by saf6260** [comment]
Isn't the point of the bot to alert users of the status of the mentor/ where/when they can receive assistance? 

TLDR, you're asking for info, imo, from the bot when you send commands. The command's response should be directed to the user who requests them, not to the lobby of users in the mentoring discord. I personally mute all channels and only receive @ notifications for specifically this discord, so any response from the bot would be lost to me

File: index.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ client.on('message', async msg => {
 		index = online_mentor_afk_list.findIndex((afk_mentor) => msg.author === afk_mentor.name)
 		if(index !== -1) {
 			online_mentor_afk_list.splice(index, 1)
-			msg.reply("has returned to their keyboard. They will be right with you :hugging::raised_hands:")
+			msg.channel.send("has returned to their keyboard. They will be right with you :hugging::raised_hands:")
```
Comment (gavrielrh) [inline index.js]:
```suggestion
			msg.channel.send(`${msg.author} has returned to their keyboard. They will be right with you :hugging::raised_hands:`)
```

Might want to test out what the message looks like 😄

Comment (galenguyer) [reply index.js]: Awesome, that does work - I'll swap everything over to that syntax as it's a lot cleaner

**Comment by galenguyer** [comment]
> TLDR, you're asking for info, imo, from the bot when you send commands. The command's response should be directed to the user who requests them

Most commands still are. Half of the changes here just remove the comma after the mention because that was bugging me 😛. Some commands have lost the ping, but bot replies don't typically need pings, as it's already clear who they're replying to. They also clog up the mentions tray for some people who use that frequently.

> I personally mute all channels and only receive @ notifications for specifically this discord, so any response from the bot would be lost to me

Given the bot replies instantly, that shouldn't be an issue

**Comment by saf6260** [comment]
> > I personally mute all channels and only receive @ notifications for specifically this discord, so any response from the bot would be lost to me
> 
> Given the bot replies instantly, that shouldn't be an issue

There have been instances where two people respond at once/bot latency slows where responses occur at random. I'll gladly show an overworked bot, @mjschuetze102 can also verify responses can slow and it can become impossible to see who a bot is talking to 

The comma is a worthy remove, and I appreciate the info, I'm just concerned with the condition that responses are sent without a notification to whom the target is when we're trying to keep people's attention. Its generally why the bot was written with the `.reply` for all messages. The more alert mentees/mentors are, the faster service can occur

**Comment by mjschuetze102** [comment]
> @mjschuetze102 can also verify responses can slow and it can become impossible to see who a bot is talking to

Oof. Why you have to call me out like that XD But yes, response times can be a pain to work with

**Comment by galenguyer** [comment]
Because of the concern about delayed replies, I've added pings in nicer-to-read locations for all the commands back! Let me know if any don't look good to you.

File: index.js [hunk 2]
```diff
@@ -129,9 +129,9 @@ client.on('message', async msg => {
 	// Non-specific commands
 	 if (msg.content.toLowerCase().startsWith("!ping")) {
 		if (online_mentor_afk_list.length < online_role.members.keyArray().length) {
-			msg.reply(`is requesting mentoring assistance ${online_role}`)
+			msg.channel.send(`<@!${msg.author.id} is requesting mentoring assistance ${online_role}`)
 		} else {
-			msg.reply("Hello, the online mentor(s) are currently away from their computer. Please be patient, they will be right back.");
+			msg.channel.send(`${msg.author}, the online mentor(s) are currently away from their computer. Please be patient, they will be right back.`);
```
Comment (mjschuetze102) [inline index.js]: Do each of these syntaxes do the same thing? If so change to use same syntax

Comment (galenguyer) [reply index.js]: Yup, those do the same. That's my bad, missed the first one

**Review by saf6260 — APPROVED** [review]
Thanks for this update. I understand its a bit different than its original intention, but the cleaned up responses are really nice. The addition is greatly appreciated
